package.json

```diff
@@ -16,6 +16,8 @@
     "eslint": "eslint --ext .tsx,.js ./src/",
     "lint": "npm run eslint",
     "format": "prettier --check --write ./src/**/*.{tsx,ts}",
+    "type-check": "tsc --noEmit",
+    "ci": "npm run type-check && npm run lint && npm run test",
```

Comment:
Can we name it something like `ci-checks` or `ci-tests`?

Reply:
Sure